map.json

```diff
@@ -34,10 +34,15 @@
 		"path" : "src/windows/win-ignoreAllFailures.ps1",
 		"description" : "Ignore errors if there is a failed boot, failed shutdown, or failed checkpoint. The computer will attempt to boot normally after an error occurs."
 	},
-	{
-		"id" : "linux-alar-fki",
-		"path" : "src/linux/linux-alar-fki.sh",
-		"description" : "alar-fki allows to recover a failed VM. Three recover scenarios are possible: fstab, initrd and kernel. NOTE: use option --run-on-repair. Se the docu for more details: https://github.com/Azure/repair-script-library/blob/master/src/linux/common/helpers/alar/README.md"
+        {
+        	"id" : "linux-alar-fki",
+        	"path" : "src/linux/linux-alar-fki.sh",
+        	"description" : "DEPRECATED: It is recommended to use linux-alar2 instead. alar-fki allows to recover a failed VM. Three recover scenarios are possible: fstab, initrd and kernel. NOTE: use option --run-on-repair. See the docu for more details: https://github.com/Azure/repair-script-library/blob/master/src/linux/common/helpers/alar/README.md"
+    	},
+	{
+		"id" : "linux-alar2",
+		"path" : "src/linux/linux-alar2.sh",
+		"description" : "alar2 allows to recover a failed VM. Various actions are available like: fstab, initrd and kernel. NOTE: use option --run-on-repair. See the docu for more details: https://github.com/Azure/repair-script-library/blob/master/src/linux/common/helpers/alar2/README.md. Please be aware that alar2 needs to be build on the recovery VM first. This takes about 2min. Please be patient."
```

Comment:
1) Remove "Please be patient." 
2) Please run alar2 on a repair vm with the az vm repair run --run-on-repair command.